src/styles/main.scss

```diff
@@ -1,3 +1,39 @@
-body {
+@import "./blocks/header.scss";
+@import "./blocks/card.scss";
+@import "../utils/variables.scss";
+@import "./blocks/stars.scss";
+@import "./blocks/price.scss";
+@import "./blocks/button.scss";
+
+* {
+  box-sizing: border-box;
   margin: 0;
+  border: none;
+}
+
+.container {
+  display: grid;
+  grid-template-columns: repeat(4, 200px);
+  justify-content: center;
+  gap: 46px 48px;
+  padding-bottom: 72px;
+  padding-top: 50px;
+}
+
+@media (max-width: 1023px) {
+  .container {
+    grid-template-columns: repeat(3, 200px);
+  }
+}
+
+@media (max-width: 767px) {
+  .container {
+    grid-template-columns: repeat(2, 200px);
+  }
+}
+
+@media (max-width: 487px) {
+  .container {
+    grid-template-columns: repeat(1, 200px);
+  }
```

Comment:
Create a variable for container size, it can be places in this file since you only use it here.